dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/fat/FATSuite.java

```diff
@@ -21,7 +21,8 @@
 	AcmeSimpleTest.class,
 	AcmeURISimpleTest.class,
 	AcmeBoulderSimpleTest.class,  
-	AcmeCaRestHandlerTest.class, 
+		AcmeCaRestHandlerTest.class,
+		AcmeSwapDirectoriesTest.class,
```

Comment:
Unnecessary white space.

```suggestion
	AcmeSwapDirectoriesTest.class,
```